src/styles/index.scss

```diff
@@ -1,3 +1,11 @@
 body {
   margin: 0;
+  height: 100vh;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
 }
+
+@import './variables';
```

Comment:
According to the code style rules, imports should be at the top of the file, before any other rules or styles are defined.